apps/nextjs-website/src/app/profile/chatbot-history/page.tsx

```diff
@@ -17,6 +18,11 @@ const ChatbotHistory = () => {
     useChatbot(true);
   const router = useRouter();
 
+  useEffect(() => {
+    getSessionsByPage(1);
+    // eslint-disable-next-line react-hooks/exhaustive-deps
+  }, []); // Needs to run only once
+
```

Comment:
You should remove this code to avoid an infinite loop:
`  useEffect(() => {
    getSessionsByPage(1);
  }, [getSessionsByPage]);`

[screencast-localhost_3000-2024_10_07-15_08_35.webm](https://github.com/user-attachments/assets/8844beb6-008d-4b9a-9ea9-cbc4af65caea)